projects/prdt/index.js

```diff
@@ -55,14 +67,31 @@ const config = {
       ADDRESSES.polygon.WETH,
     ],
   },
-  //  nibiru: {
-  //    owners: Object.values({
-  //      predictionPROV3: "0x062EB9830D1f1f0C64ac598eC7921f0cbD6d4841",
-  //    }),
-  //    tokens: [
-  //      ADDRESSES.null,
-  //    ],
-  //  },
+  nibiru: {
+    owners: Object.values({
+      // Owner from constructor of https://nibiscan.io/address/0x062EB9830D1f1f0C64ac598eC7921f0cbD6d4841/contract/6900/code
+      predictionPROOwner: "0x4cbB5d1c808F0FEe11575E77025b40f602793286",
+      predictionPROV3: "0x062EB9830D1f1f0C64ac598eC7921f0cbD6d4841",
+    }),
+    tokens: [
+      ADDRESSES.null,
+      ...Object.entries(ADDRESSES.nibiru).filter(([symbol, addr]) => {
+        const ok = addr.startsWith("0x") && addr.length === 42
+        console.debug("DEBUG %o: ", { ok, symbol, addr })
+        return ok
+      }).map(([_symbol, addr]) => addr),
+      // ADDRESSES.nibiru,
+    ],
+    // On Ethreeum, the native asset is merged with WETH automatically.
+    // For Nibiru, the native token balance is WNIBI. It has to be WNIBI and not
+    // NIBI because the EVM uses 18 decimals.
+    transformAddress: (addr) => {
```

Comment:
this case has been handled here https://github.com/DefiLlama/defillama-server/pull/9904 
we prefer any address mappings and pricing logic to be in the server repo

Reply:
Thanks, updated the file for re-review, removing the unnecessary bits.